pkg/etcd/client.go

```diff
@@ -142,3 +151,37 @@ func (c *Client) Health(ctx context.Context) error {
 	return err
 
 }
+
+// Write tries to write value with a given key and returns indicator if write operation succeed
+func (c *Client) Write(ctx context.Context, key string, value string, ttl int64) (bool, error) {
+
+	leaseResp, err := c.client.Lease.Grant(ctx, ttl)
+
+	if err != nil {
+		return false, fmt.Errorf("can't get TTL lease: %w", err)
+	}
+
+	txnResp, err := c.client.KV.Txn(ctx).If(
+		notFound(key),
```

Comment:
I reckon this is some "write if absent" thing? Why don't we always overwrite the value? If we want to keep this behavior, I'd suggest to rename the method to `Create`, `PutIfAbsent` or similar and add a note about this to the godoc of the method.

Reply:
Hm, but why? the idea is just to check, if communication between kine and non-kine worlds works, why to complicate it? I did the writing to the "etcd" in the easiest possible way, mostly because kine etcd api implementation is limited (see kine source code).

Reply:
So I checked with the history, if I do not use "notFound" guard, the underlying lib goes with etcd PUT method which is not implemented in kine.

Reply:
> So I checked with the history, if I do not use "notFound" guard, the underlying lib goes with etcd PUT method which is not implemented in kine.

Ah, that makes perfect sense then! Might be worthwhile to add this as a comment here.